tensorflow/tools/ci_build/linux/mkl/build-dev-container.sh

```diff
@@ -147,13 +159,24 @@ function build_container()
     TF_DOCKER_BUILD_ARGS+=("--build-arg OPENMPI_VERSION=${OPENMPI_VERSION}")
     TF_DOCKER_BUILD_ARGS+=("--build-arg OPENMPI_DOWNLOAD_URL=${OPENMPI_DOWNLOAD_URL}")
     TF_DOCKER_BUILD_ARGS+=("--build-arg HOROVOD_VERSION=${HOROVOD_VERSION}")
+    TF_DOCKER_BUILD_ARGS+=("--build-arg INSTALL_HOROVOD_FROM_COMMIT=${INSTALL_HOROVOD_FROM_COMMIT}")
+    TF_DOCKER_BUILD_ARGS+=("--build-arg BUILD_SSH=${BUILD_SSH}")
   fi
 
   # Add build arg --nightly_flag for the nightly build
   if [[ ${IS_NIGHTLY} == "yes" ]]; then
     TF_DOCKER_BUILD_ARGS+=("--build-arg TF_NIGHTLY_FLAG=--nightly_flag")
   fi
 
+  # Add build arg GCC8 install
+  if [[ ${ENABLE_GCC8} == "yes" ]]; then
+    TF_DOCKER_BUILD_ARGS+=("--build-arg ENABLE_GCC8=${ENABLE_GCC8}")
+  fi
+
+  if [[ ${RELEASE_CONTAINER} == "yes" ]]; then
+    TF_DOCKER_BUILD_ARGS+=("--build-arg RELEASE_CONTAINER=${RELEASE_CONTAINER}")
+  fi
```

Comment:
Nit: Same here. We don't need `if` since the Docker file already has one.
```suggestion
  TF_DOCKER_BUILD_ARGS+=("--build-arg RELEASE_CONTAINER=${RELEASE_CONTAINER}")
```

Reply:
testing these will update